OpenTabletDriver.UX.MacOS/Program.cs

```diff
@@ -7,6 +8,18 @@ class Program
         [STAThread]
         public static void Main(string[] args)
         {
+            var baseDirectory = AppDomain.CurrentDomain.BaseDirectory;
+
+            if (baseDirectory.StartsWith("/private/var/folders"))
+            {
+                _ = new Application(Eto.Platforms.Mac64);
+                MessageBox.Show(
+                    "OpenTabletDriver cannot be run from this folder. " +
+                    "Please move it to elsewhere, " +
+                    "such as to the Applications folder.");
```

Comment:
```suggestion
                    "OpenTabletDriver cannot be run from this folder (/private/var/folders). " +
                    "Please move it elsewhere, such as to the Applications folder.");
```
Rewording and clarification.

Reply:
The path /private/var/folders is invisible to the user.

From the users point of view, the app is still in Downloads folder. 

The system silently copy the app to that folder and launch from there if the app is downloaded from internet.

Reply:
Ah, I see, I'm not really familiar with macOS.
Probably still worth considering clarifying it without the path though:
```suggestion
                    "OpenTabletDriver cannot be run from this folder for security reasons. " +
                    "Please move it elsewhere, such as to the Applications folder.");
```